recipes/strec/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.0" %}
+
+package:
+  name: strec
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/strec/archive/{{ version }}.tar.gz
+  sha256: 5eb846bb460dd02959baf5bce0b09ae2b7ec32329f7b804c749cc2d1fe91a830
+
+build:
+  number: 0
+  skip: True  # [py27]
```

Comment:
Once we find out how to [deal with scripts](https://github.com/usgs/strec/blob/master/setup.py#L16) we can make this `noarch`.